docs/standards/introduction.md

```diff
@@ -5,32 +5,38 @@ sidebar_position: 1
 
 # The LUKSO Standard Proposals (LSPs)
 
-LUKSO comes with a set of new standards for smart contracts and Dapp development. These new standards are based around [Universal Profiles](./universal-profile/introduction.md) and [NFT2.0](./nft-2.0/01-introduction.md).
+---
```

Comment:
I would prefer to be consistent with formatting.

Pages currently don't use an horizontal line after the title, why would this page have an horizontal line and not the other ones?

I recommend to not add this `---` (same for other pages in this PR)